.rspec

```diff
@@ -1 +1,3 @@
--fd --color
+--format documentation
+--color
+--order defined
```

Comment:
Well, I had titled the specs such to read properly when run in documentation mode, but running them in random order may be a wise choice. Done.